media-autobuild_suite.bat

```diff
@@ -1581,11 +1581,10 @@ if exist %build%\compilation_failed del %build%\compilation_failed
 if exist %build%\fail_comp del %build%\compilation_failed
 
 REM Test mklink availability
-set symlinkSupport=""
```

Comment:
Didn't even notice this was mistyped, heh.

Reply:
same, I only noticed it when I realized it's better to just go ahead and use the MSYS variable